lib/host_config.ex

```diff
@@ -0,0 +1,64 @@
+defmodule RemoteDockers.HostConfig do
+  @enforce_keys [:hostname, :port]
+  defstruct [:hostname, :port, :ssl]
+
+  @doc """
+  Build configuration with defaults
+
+  default:
+  ```
+  hostname: "localhost"
+  port: 2376
+  ```
+  """
+  def build() do
+    %RemoteDockers.HostConfig{
+      hostname: "localhost",
+      port: 2376
```

Comment:
Couldn't we put this default port value as a \@constant ?

Reply:
don't change anything, it's used only here.
I use @constant if it's used many times.